pkg/config/engine/containerd/option.go

```diff
@@ -21,16 +21,13 @@ import (
 	"github.com/NVIDIA/nvidia-container-toolkit/pkg/config/toml"
 )
 
-const (
-	defaultRuntimeType = "io.containerd.runc.v2"
-)
-
 type builder struct {
 	logger               logger.Interface
 	configSource         toml.Loader
+	configVersion        int
```

Comment:
Did we want to allow the `CRIRuntimePluginName` to be overridden?

Reply:
I didn't update this to allow it to be overridden. If we see this as a requirement, plumbing a value through should be relatively straightforward.